test/fixtures/boilerplate-visibility.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Released AMP components</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+  <style>body{-webkit-animation:-amp-start 1s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 1s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 1s steps(1,end) 0s 1 normal both;animation:-amp-start 1s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
```

Comment:
One note to make here. Ideally I'd like to tag this style with some attribute, e.g. `<style amp-starter>` or such. I'd prefer this to our current system where we do the opposite and use `<style amp-custom>` for the user-defined styles. Boilerplate is intended to be copy-pasted into HTML and thus extra few chars don't hurt. Then, of course, I'd like not to have it at all. Please include as a separate section in our doc and call it out directly for the validator bug.